narwhals/_dask/group_by.py

```diff
@@ -36,14 +36,38 @@ def agg(s0: pd.core.groupby.generic.SeriesGroupBy) -> int:
     )
 
 
+def var(
+    ddof: int = 1,
+) -> Callable[
+    [pd.core.groupby.generic.SeriesGroupBy], pd.core.groupby.generic.SeriesGroupBy
+]:
+    from functools import partial
+
+    import dask.dataframe as dd
+
+    return partial(dd.groupby.DataFrameGroupBy.var, ddof=ddof)
```

Comment:
dask nightly build seems to fail to access `dd.groupby.DataFrameGroupBy`, I will investigate further